features/coffeescript_compilation_error.feature

```diff
@@ -3,6 +3,12 @@ Feature: Coffeescript compilation error
   As a developer with an error in my coffeescript file
   I want an appropriate error message
 
+  Background:
+    Given I have configured "filePattern" to be "**/*.coffee"
+    And I have configured "transpilers" to contain
+      | extension | module        |
```

Comment:
FYI LiveScript provides a nice way to do this using a mix of object and array comprehensions: https://github.com/Originate/exosphere-users-service/blob/master/features/steps/steps.ls#L38